src/core/Notification/InlineNotification/InlineNotification.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { InlineNotification } from './InlineNotification';
+import { axeTest } from '../../../utils/test';
+
+describe('props', () => {
+  describe('children', () => {
+    const notificationWithElementChild = (
+      <InlineNotification labelText="Lorem ipsum dolor sit">
+        <div>Test notification</div>
+      </InlineNotification>
+    );
+    it('has the given content', () => {
+      const { container } = render(notificationWithElementChild);
+      expect(container.firstChild).toHaveTextContent('Test notification');
+    });
+
+    it('should match snapshot', () => {
+      const { container } = render(notificationWithElementChild);
+      expect(container).toMatchSnapshot();
+    });
+  });
+  describe('labelText className Test', () => {
+    const InlineInlineNotificationComponent = (
+      <InlineNotification labelText="Inline Notification label">
+        Testcontent
+      </InlineNotification>
+    );
+    it('Inline component should contain given labelText', () => {
```

Comment:
We could improve readability and remove unnecessary repetition from test name, so it could be shortened to be e.g:

```
it('should contain given labelText', () => {
```

Reply:
This seems to be now fixed.